app/utils/notification/index.ts

```diff
@@ -90,12 +90,22 @@ export const emitNotificationError = (type: 'Team' | 'Channel' | 'Post' | 'Conne
 
 export const scheduleExpiredNotification = (serverUrl: string, session: Session, serverName: string, locale = DEFAULT_LOCALE) => {
     const expiresAt = session?.expires_at || 0;
-    const expiresInDays = Math.ceil(Math.abs(moment.duration(moment().diff(moment(expiresAt))).asDays()));
+    const expiresInHours = Math.ceil(Math.abs(moment.duration(moment().diff(moment(expiresAt))).asHours()));
+    const expiresInDays = Math.floor(expiresInHours / 24); // Calculate expiresInDays
+    const remainingHours = expiresInHours % 24; // Calculate remaining hours
     const intl = createIntl({locale, messages: getTranslations(locale)});
-    const body = intl.formatMessage({
-        id: 'mobile.session_expired',
-        defaultMessage: 'Please log in to continue receiving notifications. Sessions for {siteName} are configured to expire every {daysCount, number} {daysCount, plural, one {day} other {days}}.',
-    }, {siteName: serverName, daysCount: expiresInDays});
+    let body = '';
+    if (expiresInDays === 0) {
+        body = intl.formatMessage({
+            id: 'mobile.session_expired',
+            defaultMessage: 'Please log in to continue receiving notifications. Sessions for {siteName} are configured to expire every {hoursCount, number} {hoursCount, plural, one {hour} other {hours}}.',
+        }, {siteName: serverName, hoursCount: remainingHours});
+    } else {
+        body = intl.formatMessage({
+            id: 'mobile.session_expired',
+            defaultMessage: 'Please log in to continue receiving notifications. Sessions for {siteName} are configured to expire every {daysCount, number} {daysCount, plural, one {day} other {days}} and {hoursCount, number} {hoursCount, plural, one {hour} other {hours}}.',
```

Comment:
In order to avoid the case of `expire every 2 days and 0 hours`, I think we can omit the hours on this part.

Reply:
You mean, when days are non zero then no nee to show hours?

Reply:
Yes, that is what I mean. But better let's wait for input from @esethna

Reply:
Makes sense to me

Reply:
Doni need to do changes in this PR or it's in testing phase now. Sorry i am not clear.

Reply:
@rashmibharambe yes, do the changes in this PR.

Reply:
@esethna @larkox - changed done, please review..